data/mods/Magiclysm/Spells/animist.json

```diff
@@ -381,5 +381,36 @@
     "base_energy_cost": 6,
     "energy_source": "HP",
     "difficulty": 6
+  },
+  {
+    "id": "rupture",
+    "type": "SPELL",
+    "name": "Rupture",
+    "description": "You pour mana into an unwilling and unprepared vessel, causing cellular break-down, internal hemorrhaging, ley lines becoming muddled, and electronics overcharging.  Immensely gruesome, crude, and was once illegal world-wide, not that it matters now.",
+    "//": "rewrited it, so now it low-damage dot spell that transform to high-damage without dot",
```

Comment:
```suggestion
```
this kind of thing belongs in the commit description, not as a note in the spell. someone looking at the json months from now won't know the significance of it being rewritten, nor would it actually matter